platoon/mpi_util.py

```diff
@@ -34,9 +34,9 @@ def launch_mpi_workers(workers_count, experiment_name, worker_args):
     args = shape_args(experiment_name, worker_args, "worker")
     info = MPI.Info.Create()
     info['host'] = socket.gethostname()
-    #  info['ompi_non_mpi'] = 'true'
     env = dict(os.environ)
-    info['env'] = '\n'.join(env.keys())
+    keep_list = ['THEANO_FLAGS', 'PLATOON_TEST_WORKERS_NUM', '_']
+    info.Set(key='env', value='\n'.join('%s=%s' % (key, env[key]) for key in keep_list if key in env))
```

Comment:
The variable `_` is `platoon-launcher`. You can refer to [this issue](https://bitbucket.org/mpi4py/mpi4py/issues/62/does-mpiinfo-only-support-built-in-keys) on MPI4PY to see my environmental variables. Of course we can do it without passing the environmental variables explicitly, I'll check that today to see how to change the codes.